numpy/core/src/multiarray/array_method.c

```diff
@@ -580,9 +611,141 @@ boundarraymethod__resolve_descripors(
 }
 
 
+static PyObject *
+boundarraymethod__simple_strided_call(
+        PyBoundArrayMethodObject *self, PyObject *arr_tuple)
+{
+    PyArrayObject *arrays[NPY_MAXARGS];
+    PyArray_Descr *descrs[NPY_MAXARGS];
+    PyArray_Descr *out_descrs[NPY_MAXARGS];
+    ssize_t length = -1;
+    int aligned = 1;
+    npy_intp strides[NPY_MAXARGS];
+    int nin = self->method->nin;
+    int nout = self->method->nout;
+
+    if (!PyTuple_CheckExact(arr_tuple) ||
+        PyTuple_Size(arr_tuple) != nin + nout) {
+        PyErr_Format(PyExc_ValueError,
+                "_simple_strided_call() takes exactly one tuple with as many "
+                "arrays as the method takes arguments (%d+%d).", nin, nout);
+        return NULL;
+    }
```

Comment:
Coverage thinks the error condition needs a test

Reply:
I didn't bother adding tests for this (at least for now) private API yet. But can add it here.